### PR TITLE
Fix workflow with new system

### PR DESCRIPTION
**Pull request by PancakeTAS**
This pull request _should_ fix the workflow. If it for some reason doesn't work first try which I highly doubt (copium) then I guess it's time to test this somewhere else.

### CODE REVIEW

**Comment by ScribbleTAS** [comment]
@MCPfannkuchenYT So the reason the old workflow was there, is that it worked with pull requests... Prs are read only by default and can't use the publish action... thats why the test report was published as an artifact, then afterwards downloaded and applied... You might be able to get the old workflow back...

**Comment by PancakeTAS** [comment]
Huh? The second workflow also runs on the PR and both PRs publish something... Let's just figure out what happens I guess?

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,11 +1,13 @@
-name: test
+name: Test Discombobulator
 
 on:
```
Comment (ScribbleTAS) [inline .github/workflows/test.yml:3]: add `on: push` then the branch in questionto this. This will run the same workflow as on pr

**Review by ScribbleTAS — CHANGES_REQUESTED** [review]
Revert everything and just add on push. That way the same workflow works for both prs and pushes

File: .github/workflows/test.yml (hunk 1 above)
Comment (PancakeTAS) [reply .github/workflows/test.yml:3]: I don't want the workflow to run on every push. It runs when you open a PR (even just draft) to develop which will then again run it for every commit as well

Comment (ScribbleTAS) [reply .github/workflows/test.yml:3]: Alright?

**Review by ScribbleTAS — APPROVED** [review]

**Pull request merged**